src/resizable_array.jl

```diff
@@ -241,7 +244,7 @@ end
 function lastwithindex(array::ResizableArray{T, V, N}) where {T, V, N}
     for index in reverse(CartesianIndices(reverse(size(array)))) #TODO improve performance of this function since it uses splatting
```

Comment:
I think the `TODO` comment is old, the splatting by itself does not hurt performance, but wrong usage of splatting does. Here everything should be fine imo